package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
-    "object-assign": "^3.0.0"
+    "object-assign": "^3.0.0",
+    "object.entries": "^1.0.4"
```

Comment:
Would prefer to use `Object.keys()` as mentioned before rather than pulling in another lib.